### PR TITLE
feat(content) More Snippets

### PR DESCRIPTION
**Pull request by Empyrean-Heaven**
#### Summary
Content "More Snippets"

#### Purpose of change
Following up on my previous PR, this time I'll be adding more grave_labels, graffiti, and survivor_notes.

#### Describe the solution
Adding more snippets.

#### Describe alternatives you've considered
Leaving it be.

#### Testing
JSON only.

#### Additional context
Hope everyone lives a wonderful life. :D

### CODE REVIEW

File: data/json/snippets/graffiti.json [hunk 1]
```diff
@@ -110,7 +145,21 @@
...
-      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" }
+      { "id": "shelter_graffiti_35", "text": "Remember <given_name>" },
+      {
+        "id": "shelter_graffiti_36",
+        "text": "Two homeless men were going at it while a zombie was eating a woman alive right next to them.  They even invited the zombie to join them."
+      },
+      {
+        "id": "shelter_graffiti_37",
+        "text": "A pregnant woman said that one of those undead fuckers fucked her in the fucking fuck fuck.  Fuck.  I had sloppy seconds.  It was worth it."
+      },
```
Comment (Karol1223) [inline data/json/snippets/graffiti.json:156]: Like a solid half of those reek of extremely poor middle school levels of humor. I don't think I've ever been less of a fan of a PR.

**Comment by KeremBabaG** [comment]
yeah I agree with the other guy
that one in particular is reddit tier humor

**Comment by KeremBabaG** [comment]
but I love the mininuke one lol

**Comment by Zireael07** [comment]
More snippets is a good idea
Middle school humor is a bad idea

**Comment by Empyrean-Heaven** [comment]
I'll remove the ones I added that were not my original writing. :D

**Comment by Karol1223** [comment]
I don't know why you're requesting my review here. You asked yesterday on the discord about whether your snippets are good or too much - you were told they're too much. Several people, myself included, have told you our stances on adding blatant memes into the game, yet you have several snippets whose sole purpose is to be subpar memes. If you were hoping for me to approve the PR, sorry, but I will not.

**Pull request closed without merging**